### DISCUSSION TITLE
Jittering when resizing rotated structures

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#13644] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While resizing a 45 degree rotated "Large Horizontal Inside Wall", I observed it jittering.

![Barotrauma_YN2XjmwdXb](https://github.com/Regalis11/Barotrauma/assets/67431770/f60a8eb9-3d33-4746-bff9-0bce0e1796d2)

### Reproduction steps

1. Place a rotatable structure in the Submarine Editor.
2. Rotate it any amount.
3. Resize it back and forth.
4. Observe the sprite and hitbox jittering randomly.

### Bug prevalence

Just once

### Single player or multiplayer?

Other

### -

Submarine Editor

### Version

v1.2.1.0 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by juanjp600** [comment]
Should be fixed in https://github.com/Regalis11/Barotrauma-development/commit/b72cec071da05e7fd4c91966dcb5c59c1024a869. The changes required are very extensive (>1000 lines changed) so it probably won't make it into the next update.
